tests/src/test/scala/system/basic/WskBasicTests.scala

```diff
@@ -492,6 +492,21 @@ class WskBasicTests
             wsk.action.get(name).stdout should not include (""""code"""")
     }
 
+    it should "create an action and limit the length of req body in the debug to 1000 bytes" in {
+            val file = Some(TestUtils.getTestActionFilename("largeFile.js"))
+            val actionName = "largeFileAction"
+            val msg = "Req Body excedes 1000 bytes and will be truncated"
+            val output = "fWbADPnp\"" //   Ending substring of truncated output
+
+            try {
+                val action = wsk.action.create(name = actionName, artifact = file, expectedExitCode = SUCCESS_EXIT, debug = Some(true)).stdout
+                action should include(output)
+                action should include(msg)
+            } finally {
+                val delAction = wsk.action.delete(name = actionName, expectedExitCode = DONTCARE_EXIT)
```

Comment:
Use the built-in cleaner instead. We would need the exit code to be successful value. Otherwise, some other test may try to create an action with the same name you are using and fail if this action wasn't actually deleted.